### PR TITLE
feat: board page

### PR DESCRIPTION
**Pull request by matheusaraujo**

### CODE REVIEW

**Review by rpidanny — APPROVED** [review]

File: src/App.css [hunk 1]
```diff
@@ -1,21 +1,11 @@
...
-  min-height: 100px;
+body {
+  overlay: hidden;
+  /* background-color: #89cdbe; */
 }
 
+@media (max-width: 600px) {
+  .some-class {
+    font-size: 14px;
+    /* Example for small screens */
```
Comment (matheusaraujo) [inline src/App.css:9]:
```suggestion
    /* Example for small screens */
```

File: src/App.tsx [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
+import Game from "./pages/Game/Game";
+import Home from "./pages/Home/Home";
+import NoPage from "./pages/NoPage/NoPage";
+
+const getBasename = () => {
+  if (window.location.hostname === "geo-quest.github.io") {
+    return "/travel-bingo";
+  }
+  return "/";
+};
```
Comment (matheusaraujo) [inline src/App.tsx:14]: nitpick: maybe we can solve that outside the code, so the application doesn't need to know anything about the environment, maybe we can use some scripts or environment variables. 

no rush on that, just a note.

File: src/games/NLAug24/Challenges.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.header-card {
+    width: 20%;
+    text-align: center;
+    font-size: 1.5em;
+    font-weight: bold;
+    padding: 10px;
+    background-color: "#f0f0f0";
+}
```
Comment (matheusaraujo) [inline src/games/NLAug24/Challenges.css:8]: suggestion: we can remove this file

File: src/games/NLAug24/Challenges.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
+/* eslint-disable arrow-parens */
```
Comment (matheusaraujo) [inline src/games/NLAug24/Challenges.tsx:1]: note to myself: I need to find a better solution for this

**Pull request merged**